registry/components/magicui/gauge-circle.tsx

```diff
@@ -0,0 +1,130 @@
+import { cn } from "@/lib/utils";
+
+interface Props {
+  max: number;
+  value: number;
+  min: number;
+  gaugePrimaryColor: string;
+  gaugeSecondaryColor: string;
+  className?: string;
+}
+
+/*
+  This is the css that should be added to a css file
+ */
+const css = `.gauge_circle {
```

Comment:
can we just convert this to inline css variables + tailwindcss?

You already started the process below a bit but looks like we didn't finish it

Reply:
Hi @dillionverma,

Yes, we can convert the CSS to inline CSS variables and use TailwindCSS.